gen/libdir/forbidden.conf

```
@@ -45,11 +45,161 @@
+# let ca = fr | co
+# let Exp-obs =
+#   [Exp & M]; rf & ext; [Exp & M]
+#   | [Exp & M]; ca & ext; [Exp & M]
+-safe Rfe Fre Coe
+
+#let Exp-haz-ob = [Exp & R]; (po & same-loc); [Exp & R]; (ca & ext); [Exp & W]
+-safe [PosRR Coe] [PosRR Fre]
```

```suggestion
-safe [PosRR Fre]
```
`[PosRR Coe]` doesn't make sense `Coe` starts (and ends) with a write.